version_config.cmake

```diff
@@ -150,6 +156,13 @@ else()
 
   set(DEFAULT_ACE_URL https://github.com/paskino/libace-conda)
   set(DEFAULT_ACE_TAG origin/master)
+  
+  # CCPi CIL
+  set(DEFAULT_CCPi-Framework_URL https://github.com/vais-ral/CCPi-Framework.git)
+  set(DEFAULT_CCPi-Framework_TAG origin/master)
```

Comment:
really `master`?

Reply:
I'm removing it in favour of a specific tag.